nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
```

Comment:
@janhahne We could implement the check explicitly, but that would probably be overdoing it for a problem that for all we know cannot happen. So I am fine without the check.